src/app/core/filter/filter-generator/filter-generator.service.spec.ts

```diff
@@ -138,19 +129,15 @@ describe("FilterGeneratorService", () => {
   });
 
   it("should create an entity filter", async () => {
-    const school1 = new TestEntity();
+    const [school1, school2] = [new TestEntity(), new TestEntity()];
     school1.name = "First School";
-    const school2 = new TestEntity();
     school2.name = "Second School";
     await TestBed.inject(EntityMapperService).saveAll([school1, school2]);
     const csr1 = new ChildSchoolRelation();
     csr1.schoolId = school1.getId();
     const csr2 = new ChildSchoolRelation();
     csr2.schoolId = school2.getId();
-    const csr3 = new ChildSchoolRelation();
-    csr3.schoolId = school2.getId();
-    const csr4 = new ChildSchoolRelation();
-    csr4.schoolId = school1.getId();
+    const [csr3, csr4] = [csr2, csr1];
```

Comment:
I am not sure this is equivalent to the previous code? Doesn't this just assign the object references now instead of a new object / deep copy?